### PR TITLE
Courses on Computer Networks and Distributed Computing added in the Malayalam language section

### PR DESCRIPTION
**Pull request by itsabinabraham**
## What does this PR do?
Courses in Computer Networking and Distributed Computing in Malayalam added

### Why is this valuable (or not)?
Playlist on relevant subjects.

### How do we know it's really free?
Videos are available as a playlist for free on YouTube

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-ml.md [hunk 1]
```diff
@@ -6,7 +6,9 @@
 * [C++](#cpp)
 * [C#](#csharp)
 * [Compiler Design](#compiler-design)
+* [Compiler Networking](#computer-networking)
```
Comment (eshellman) [inline courses/free-courses-ml.md:9]: compiler networking?????

File: courses/free-courses-ml.md [hunk 2]
```diff
@@ -66,11 +68,22 @@
 * [Compiler Design](https://www.youtube.com/playlist?list=PLwe8bwPXjlLtPlbzxU_SICYI3PeYpEAtJ) -  Learn CSE Malayalam
 
 
+### Computer Networking
```
Comment (eshellman) [inline courses/free-courses-ml.md:71]: just "Networking"

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-ml.md (hunk 2 above)
Comment (itsabinabraham) [reply courses/free-courses-ml.md]: Done.

File: courses/free-courses-ml.md (hunk 1 above)
Comment (itsabinabraham) [reply courses/free-courses-ml.md]: Done.

File: courses/free-courses-ml.md [hunk 3]
```diff
@@ -177,6 +184,12 @@
 * [SQL / MYSQL Malayalam Tutorial Full Course](https://www.youtube.com/playlist?list=PL1JrLEBAapUXMuDKwPVVzhlBge9fdxV51) - Yes Tech Media
 
 
+### Networking
+
+* [Computer Networks - S5](https://www.youtube.com/playlist?list=PLI74-7rtCb9BRUw6JzCm_wwLdQJc3jSwO) -  Eduline CSE Knowledge Sharing Platform
```
Comment (eshellman) [inline courses/free-courses-ml.md:189]: could you switch the order of these two to make it alphabetical. Our linter is not checkingg malalayam.

Comment (itsabinabraham) [reply courses/free-courses-ml.md]: Resolved.

**Pull request merged**